src/tests/ftest/aggregation/basic.yaml

```diff
@@ -28,8 +28,7 @@ ior:
   env_vars:
     - D_LOG_MASK=INFO
   client_processes:
-    np_12:
-      np: 12
+    ppn: 6
```

Comment:
Since the number of servers is the same, we could maybe keep the same number of total client processes. Though I'm not sure if it matters for this test.